src/test/java/driver/DriverSingleton.java

```diff
@@ -0,0 +1,33 @@
+package driver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class DriverSingleton {
```

Comment:
Singleton is mixed with Factory patterns. Please separate.
Meanwhile, the class breaks the Single Responsibility Principle because it manages browser window size. Please remove from this class